coalib/bears/requirements/GemRequirement.py

```diff
@@ -29,3 +31,31 @@ def __init__(self, package, version="", require=""):
         """
         PackageRequirement.__init__(self, 'gem', package, version)
         self.require = require
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> GemRequirement('rubocop').install_command()
+        'gem install rubocop'
+
+        >>> GemRequirement('scss_lint', '', 'false').install_command()
+        "gem install 'scss_lint', require: false"
+
+        :param return: A string with the installation command.
+        """
+        if self.require:
+            return "gem install '{}', require: {}".format(self.package,
```

Comment:
I feel like it'd be better to do something like

```
result = 'gem ...'

if ...:
    result += ...

retur result
```

this way you don't duplicate part of the string leadig to the error pointed out above